registration/views.py

```diff
@@ -22,3 +22,17 @@ def get_serializer_class(self):
         if self.action in ["retrieve", "create"]:
             return FamilyDetailSerializer
         return FamilySerializer
+
+
+class FieldViewSet(
+    viewsets.GenericViewSet,
+    mixins.ListModelMixin,
+    mixins.RetrieveModelMixin,
+    mixins.CreateModelMixin,
```

Comment:
we might not need these two for now - can we remove em and update the tests accordingly?

Reply:
ok gotcha, just listing out all the fields is good enough for now